.github/workflows/wheel_macos_x86_64.yml

```diff
@@ -24,6 +29,7 @@ concurrency:
 jobs:
   set_wheel_build_matrix:
     if: |
+      github.event.pull_request.draft == false &&
```

Comment:
# Issue
Since `&&` is higher precedence than `||`, we need to be careful here:
```yaml
      github.event_name != 'pull_request' ||
      (github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'ci:build_wheels'))
```
(Also note that `github.event_name == 'workflow_dispatch'` is actually redundant here.)